praw/__init__.py

```diff
@@ -171,6 +171,7 @@ class Config(object):  # pylint: disable=R0903
                  'subreddit_random':    'r/%s/random/',
                  'subreddit_settings':  'r/%s/about/edit/',
                  'subscribe':           'api/subscribe/',
+                 'suggested_sort':      'api/set_suggested_sort',
```

Comment:
Can you add the trailing slash here for consistency (unless it doesn't work with it)?